tests/serde_deserializer.rs

```diff
@@ -414,6 +422,8 @@ fn ipld_deserializer_tuple_struct_errors() {
     let error_not_enough = TupleStruct::deserialize(ipld_not_enough);
     assert!(error_not_enough.is_err());
 
+    /*
+    // TODO: this should work because of deny_unknown_fields but needs to be fixed upstream
```

Comment:
I have a patch that should fix this. Can you merge https://github.com/ipld/github-mgmt/pull/85 when you get up so I can push without having to make PRs on PRs?